app/assets/stylesheets/desktop/compose.scss

```diff
@@ -81,12 +81,6 @@
   }
 }
 
-.composer-popup-container {
-  max-width: 1500px;
```

Comment:
Does this mean composer stays full width on super wide windows?

Reply:
That is just a wrapper for composer-popup. (the just-in-time message that displays over the composer preview)

The removed style was unused, as the popup is its only child and is positioned absolutely, so the wrapper has no width/height.